src/Hedgehog/Gen.fs

```diff
@@ -413,36 +400,36 @@ module Gen =
         item [false; true]
 
     /// Generates a random byte.
-    let byte : Gen<byte> =
-        sizedBounded ()
+    let byte (range : Range<byte>) : Gen<byte> =
+        integral range
 
     /// Generates a random signed byte.
-    let sbyte : Gen<sbyte> =
-        sizedBounded ()
+    let sbyte (range : Range<sbyte>) : Gen<sbyte> =
+        integral range
```

Comment:
So, for example here, if I rename `integral` back to `range` the F# compiler wouldn't allow me to apply `range` to `range`.